src/types/app.d.ts

```diff
@@ -1262,7 +1262,9 @@ interface PerpsParams {
   openingFeeRate: BigNumber
 }
 
-interface Store extends CommonSlice, ModalSlice {}
+interface Store extends CommonSlice, ModalSlice {
+  withdrawFromVaults: any
+}
```

Comment:
This change was introduced by me and can be undone
```suggestion
interface Store extends CommonSlice, ModalSlice {}
```